libs/ngxtension/inject-destroy/src/inject-destroy.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import {
+	ComponentFixture,
+	TestBed,
+	fakeAsync,
+	tick,
+} from '@angular/core/testing';
+import { interval, takeUntil } from 'rxjs';
+import { injectDestroy } from './inject-destroy';
+
+describe(injectDestroy.name, () => {
+	describe('emits when the component is destroyed', () => {
+		@Component({ standalone: true, template: '' })
+		class TestComponent implements OnInit {
+			destroy$ = injectDestroy();
+			count = 0;
+
+			ngOnInit() {
+				interval(1000)
+					.pipe(takeUntil(this.destroy$))
+					.subscribe(() => this.count++);
+			}
+		}
+
+		let component: TestComponent;
+		let fixture: ComponentFixture<TestComponent>;
+
+		beforeEach(async () => {
+			await TestBed.configureTestingModule({
+				imports: [TestComponent],
+			}).compileComponents();
```

Comment:
suggestion: you can use `TestBed.createComponent()` API instead of `configureTestingModule().compileComponents()`